napari/utils/colormaps/__init__.py

```diff
@@ -4,6 +4,7 @@
     ALL_COLORMAPS,
     AVAILABLE_COLORMAPS,
     CYMRGB,
+    INVERSE_LUT,
```

Comment:
```suggestion
    INVERSE_LUTS,
```

Reply:
If it's not too much trouble, I'd suggest using `INVERSE_COLORMAPS` to be consistent with the other existing names. I know LUT vs. colormap is a preference, but it doesn't seem right to switch suddenly. Plus, [we also have a glossary](https://napari.org/stable/further-resources/glossary.html) to help with that translation.